appinfo/info.xml

```diff
@@ -13,6 +13,6 @@
         <filesystem/>
     </types>
     <dependencies>
-        <nextcloud min-version="12" max-version="13"/>
```

Comment:
I thought the app would also work for NC12?

Reply:
this was only for early testers. We didn't backported all details like blocking folders in the web interface. So with the release of Nc13 this will be the version on the app store